addons/account/tests/test_account_move_entry.py

```diff
@@ -494,8 +494,9 @@ def test_journal_sequence_ordering(self):
         self.assertEqual(copies[5].name, 'XMISC/2019/00004')
 
         # Can't have twice the same name
-        with self.assertRaises(ValidationError):
+        with self.assertRaises(psycopg2.IntegrityError), self.env.cr.savepoint(), mute_logger('odoo.sql_db'):
             copies[0].name = 'XMISC/2019/00001'
+            copies[0].flush()
```

Comment:
This line is not necessary, since `savepoint()` automatically flushes.

Reply:
It won't raise the error inside the `assertRaises` context manager if I don't put it

Reply:
Changing the order of the context managers did the trick